service/src/test/resources/ehr/mapper/observation/example-observation-resource-26.json

```diff
@@ -52,6 +52,28 @@
                 ],
                 "text": "Example interpretation text 1"
             }
+        },
+        {
+            "referenceRange": [
+                {
+                    "text": "Low only",
+                    "low": {
+                        "value":  0.1,
+                        "unit": "mg/L"
+                    }
+                }
+            ]
+        },
+        {
+            "referenceRange": [
+                {
+                    "text": "High only",
+                    "low": {
+                        "value": 0.6,
+                        "unit": "mg/L"
+                    }
+                }
```

Comment:
If these ranges are distinct from the other one above, might be nice to use different values so it's clear they're about different things.

Now we get
> [... Range: Low 0.1 mg/L High 0.6 mg/L] [Range: Low 0.1 mg/L] [Range: Low 0.6 mg/L]

which looks like repetition. If they're different, let's have the test data different. If they're the same, is the repetition appropriate?

Reply:
Yep these ranges are distinct from each other. Will update the values to to make the data more distinct.